lib/versions.txt

```diff
@@ -1,4 +1,4 @@
-formiojs@4.14.0/dist/formio.full.min.js
-formiojs@4.14.0/dist/formio.full.js
-@formio/premium@1.17.1/dist/premium.js
-@formio/premium@1.17.1/dist/premium.min.js
+formiojs@4.14.8/dist/formio.full.min.js
```

Comment:
did this miss-align with package json?

Reply:
yes, this file needs to be updaed manually and missed